src/Cake.Npm.Tests/Update/NpmUpdateToolTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Cake.Npm.Update;
+using Xunit;
+
+namespace Cake.Npm.Tests.Update
+{
+    public class NpmUpdateToolTests
+    {
+        public sealed class TheAutomatedInstallMethod
```

Comment:
Copy / Paste error? Or to which `AutomatedInstall` method does this reference

Reply:
yea, updated all the instances where this occurs